src/features/authentication/pages/login/login.ts

```diff
@@ -54,32 +48,17 @@ export class LoginPage implements OnInit, OnDestroy {
   );
 
   constructor(
-    private fb: FormBuilder,
     private appEffects: AppEffects,
     private authEffects: AuthEffects,
     private navCtrl: NavController,
     private navParams: NavParams
   ) {}
 
-  ngOnInit() {
-    this.createForm();
-  }
-
   ngOnDestroy() {
     this.userSub.unsubscribe();
     this.loginSub.unsubscribe();
   }
 
-  createForm() {
-    this.agreeForm = this.fb.group({
-      agree: ["", Validators.required]
```

Comment:
There is no need to remove the form, the problem with this is that we are starting the form with "" for a boolean input, the checkbox, so if we just change that to a false it works without changing anything more. 

I created a PR for this #55 if you want to look at it